api/app.go

```diff
@@ -72,6 +88,17 @@ func GetApp(host string, port int, configPath string, debug, fast bool, logger z
 	return app, nil
```

Comment:
Why return an error if it never errors? Why not just `NewApp(...) *App`?

Reply:
But there is an app.Configure() just before this line that can return an error. Or maybe I misunderstood the question?

Reply:
My bad, nevermind.